addons/web/static/src/legacy/js/public/public_root_instance.js

```diff
@@ -43,14 +76,17 @@ export default lazyloader.allScriptsLoaded.then(async () => {
     serviceRegistry.add("legacy_notification", makeLegacyNotificationService(legacyEnv));
     serviceRegistry.add("legacy_crash_manager", makeLegacyCrashManagerService(legacyEnv));
     serviceRegistry.add("legacy_dialog_mapping", makeLegacyDialogMappingService(legacyEnv));
-
     await Promise.all([owl.utils.whenReady(), session.is_bound]);
-    //legacyEnv.qweb.addTemplates(session.owlTemplates);
 
     const wowlEnv = makeEnv();
+    wowlEnv.qweb.addTemplates(await loadBundleTemplates("web.assets_frontend"));
```

Comment:
these templates were not loaded before?

Reply:
There was no templates in this bundle before. Now it contains the wowl notifications template, and will probably contain more things in the future